functions/log2/README.md

```diff
@@ -0,0 +1,24 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
```

Comment:
There are multiple "hello world" samples in the docs (I don't know why), which is where these samples are used. I think this particular sample will go away after the docs are re-organized.